api/config_keys.py

```diff
@@ -0,0 +1,46 @@
+from pydantic import BaseModel
+from typing import Optional
+
+
+class APP(BaseModel):
+    VideoPath: Optional[str]
+    Host: Optional[str]
+    Port: Optional[str] 
+    Resolution: Optional[str]
+    Encoder: Optional[str]
+
+class DETECTOR(BaseModel):
+     Device: Optional[str]
+     Name: Optional[str]
+     ImageSize: Optional[str]
+     ModelPath: Optional[str]
+     ClassID: Optional[str]
+     MinScore: Optional[str]
+
+class CORE_(BaseModel):
+    Host: Optional[str]
+    QueuePort: Optional[str]
+    QueueAuthKey: Optional[str]
+
+class POSTPROCESSOR(BaseModel):
+    MaxTrackFrame: Optional[str]
+    NMSThreshold: Optional[str]
+    DistThreshold: Optional[str]
+    DistMethod: Optional[str]
+
+class LOGGER(BaseModel):
+    Name: Optional[str]
+    TimeInterval: Optional[str]
+    LogDirectory: Optional[str]
+
+class API_(BaseModel):
+    Host: Optional[str]
+    Port: Optional[str]
+
+class Config(BaseModel):
+    App: Optional[APP]
+    Detector: Optional[DETECTOR]
+    PostProcessor: Optional[POSTPROCESSOR]
+    Logger: Optional[LOGGER]
+    API:  Optional[API_]
+    CORE: Optional[CORE_]
```

Comment:
I don't think its ideal for the API to receive requests in PascalCase. snake_case seems to be what some would use nowadays with a python API.

With @mats-claassen we were working on https://github.com/neuralet/neuralet/issues/25 and by looking at FastAPI and Pydantic's API I implemented with snake_case (and some validation for video_path). We also found your commit about config_keys (that's why I used the same file name)

Here's the file if you want to look at it, https://github.com/xmartlabs/smart-social-distancing/blob/227d505ca643a252a7a4667ec91e60c633e32620/api/models/config_keys.py

My idea was to rebase with this after this PR was merged, adapt it to the newly proposed changes and open a new pr proposing these changes (along with a frontend implementation to edit video path)

What do you think?

Reply:
This keys came from the config files as is in [configParser](https://docs.python.org/3/library/configparser.html) examples.
If you think it's better to follow the FastApi and Pydantic snake_case rules feel free to change it and good idea to change it after merge. thanks.